youtube_dl/YoutubeDL.py

```diff
@@ -833,6 +833,8 @@ def iter_playlistitems(format):
                                                       extra_info=extra)
                 playlist_results.append(entry_result)
             ie_result['entries'] = playlist_results
+            if result_type == 'playlist':
```

Comment:
Opening message is already applied for `multi_video`, I don't see any reason for closing message not to be applied for it as well.